lib/Devel/Cover/Report/CoverallsJSON.pm

```diff
@@ -0,0 +1,175 @@
+package Devel::Cover::Report::CoverallsJSON;
```

Comment:
is this the right way to use a perl library?  (i.e., copy/pasting it)

Reply:
Well, I just took the source code from https://metacpan.org/pod/Devel::Cover::Report::Coveralls, modified it a single line, added attribution and put it there. Alternative would have been to release our own CPAN module which I didn't bother to do due to how temporary our use of this code is intended to be.